public/javascripts/global.js

```diff
@@ -129,14 +129,15 @@ function fetchQuiz(event) {
     var target = $(event.target);
     if (target.hasClass('quiz-btn')){
       var team = $('.selected-team'),
-          id = team.data('id');
+          id = team.data('team-id'),
```

Comment:
Right now I'm just pulling the API id through bc I haven't seen an immediate use for the RB id and that would make the url really long to pull both.